Libs/MRML/Core/vtkMRMLDiffusionImageVolumeNode.h

```diff
@@ -50,7 +50,7 @@ class VTK_MRML_EXPORT vtkMRMLDiffusionImageVolumeNode : public vtkMRMLTensorVolu
 
   ///
   /// Get node XML tag name (like Volume, Model)
-  const char* GetNodeTagName() override {return "DiffusionImageVolume";}
+  const char* GetNodeTagName() override {return "vtkMRMLDiffusionImageVolumeNode";}
```

Comment:
Why this was changed ? 

None of the existing tag name corresponds to the class name:

```
$ ack --cpp "GetNodeTagName\("
[...]
Libs/MRML/Core/vtkMRMLDisplayableHierarchyNode.h
53:  const char* GetNodeTagName() override {return "DisplayableHierarchy";}

Libs/MRML/Core/vtkMRMLInteractionNode.h
25:  const char* GetNodeTagName() override {return "Interaction";}

Libs/MRML/Core/vtkMRMLdGEMRICProceduralColorNode.h
52:  const char* GetNodeTagName() override {return "dGEMRICProceduralColor";}

Libs/MRML/Core/vtkMRMLHierarchyStorageNode.h
36:  const char* GetNodeTagName() override {return "HierarchyStorage";}

Libs/MRML/Core/vtkMRMLFiducialListNode.h
74:  const char* GetNodeTagName() override {return "FiducialList";}
[...]
```

Reply:
Good catch. This change was definitely not intentional and the node tag name should be reverted to the original.